app/models/question_answer.rb

```diff
@@ -0,0 +1,4 @@
+class QuestionAnswer < ActiveRecord::Base
+  belongs_to :question
```

Comment:
That's interesting. I didn't take that scenario into account. We could choose to only allow adming to archive questions instead. That way we don't have to store question description in multiple places and we don't loose the question either.